libraries/AP_Motors/AP_MotorsTri.cpp

```diff
@@ -51,6 +51,9 @@ void AP_MotorsTri::init(motor_frame_class frame_class, motor_frame_type frame_ty
     if (frame_type == MOTOR_FRAME_TYPE_PLUSREV) {
         _pitch_reversed = true;
     }
+    else {
+        _pitch_reversed = false;
+    }
```

Comment:
```suggestion
    _pitch_reversed = frame_type == MOTOR_FRAME_TYPE_PLUSREV;
```

Reply:
Yeah, I wanted to do this, but was trying to match the code below. I'll change it.

Reply:
I would change the one below too.... 

Although it was me that added it, no idea why I did it like that.